book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
```

Comment:
- [  ] Kindly add a method for adding rental. Link to the example in a previous [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship). 
- This does the  `many-to-many` (also `has-many-through`) relationship between `Person` and `Book`

### Instruction:
![image](https://github.com/ybadr99/school_library/assets/50990925/46cb743b-6180-46f6-89b4-94736807bb1c)